### ISSUE TITLE
Publish package to `pip`

### ISSUE THREAD

**Issue by david-zwicker** [zwicker-group/py-tools#4]
We would like to publish the package to `pip` to make it easier to install. Unfortunately, the name `tools` and `py-tools` is already used on `pip`, so we need to find a better alternative and rename this package.

Options so far:
* `py-science-tools`
* `science-tools`
* `zwicker-tools`
* `py-zwicker`

Any suggestions?

**Comment by dostuffthatmatters** [comment]
You might find the setup of our utility library helpful: https://github.com/tum-esm/utils

I use [PDM](https://pdm-project.org/latest/) to build and publish releases on PyPI: `pdm build && pdm publish`.

Looking forward to more tools from your group we can reuse :D

**Comment by david-zwicker** [comment]
Yeah, I think it is a good idea to have such an utils package to get people easily started. We don't have any technical issues publishing on pip, it's rather that we currently picked a bad package name – you guys apparently did a better job :)

**Comment by dostuffthatmatters** [comment]
I think `zwicker-tools` is nice :) I think you can leave the "py" in the repo name but don't need it in the PyPI package.

**Comment by david-zwicker** [comment]
We decided to call the package `utilitiez`.
